### ISSUE TITLE
Unity doesn't show anything

### ISSUE THREAD

**Issue by JoseLucasASilva** [Unity3D-Wine-Support/Unity3D-on-Wine#79]
Hi, I tried to install Unity using PlayOnLinux, during the installation process a message is showed, and says, "Failed to enable .NET 3.5", ok, the process isn't canceled, but after, when I go to open the Unity, nothing is showed:
The first screen stay black, only this.

**Comment by NathanWarden** [comment]
The native Linux Experimental version works much better than the Wine setup ever did in my opinion.

https://forum.unity3d.com/threads/unity-on-linux-release-notes-and-known-issues.350256/

Make sure you grab the builds from the bottom of the thread, the top ones are over a year old now.

**Comment by JoseLucasASilva** [comment]
Yeah, but the problem is that it only runs in 64-bit environment, my system is 32-bit :/

**Comment by DoctorJellyface** [comment]
Then switch to a 64 bit distro. If your hardware doesn’t support it then I doubt it would be able to run Unity properly.

**Comment by NathanWarden** [comment]
I agree, almost 100% guaranteed if you have a desktop made after around 2005 your system is 64 bit. Most older laptops that have something other than an Atom processor are also 64 bit. The somewhat newer Atom CPU's are also 64 bit.

If you're not sure, just put a 64 bit distro onto a USB drive or burn it to a DVD. If it boots you have a 64 bit system. If it gives you errors, then your CPU may be 32 bit. Many 64 bit systems used to be sold with 32 bit Windows, leaving customers, including myself, thinking they have 32 bit hardware, when they are in fact 64 bit :)
